### PR TITLE
Remove `<center>` tag and Refactor code again

### PR DESCRIPTION
**Pull request by Chiroyce1**
Lots of changes

- Added a `<div>` for all pages instead of `<center>`
- Made all pages' code consistent, now all of them contains the following
   - A `<head>` tag with the `<title>` and `<link>` for the CSS
   - A `<body>` tag with the `<div>`, which contains all the contents
- Removed indentation - I don't think you like it a lot and the size and type is not similar for contributors.
- More changes may come soon, you can see the preview [here](https://chiroyce1.github.io/derter-menter.github.io/). Don't merge it yet :P

### CODE REVIEW

File: youtube.html [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-   <a href='github'><button class="b github">Github</button></a>
-    <br>
-    <img src='https://u.cubeupload.com/Cruncherbones/cooltext402658688313.png' align='middle'>
-    <p>I usually just upload videos of different football games I play. <a href=https://www.youtube.com/channel/UCKA4top0zmSO_FyoVzA9lMA/>my channel</a>.</p>
-    latest video:
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/5WWflFxUZSU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-  </center>
+<div class="container">
+
+<a href='/'><button class="b /">Home</button></a>
```
Comment (Chiroyce1) [inline youtube.html:12]: What's with the `/` for the class name? Is it intentional?

File: youtube.html [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+<a href='scratch'><button class="b scratch">Scratch</button></a>
+<a href='youtube'><button class="b yotube">YouTube</button></a>
```
Comment (Chiroyce1) [inline youtube.html:14]: Typo for `yotube`, it may be confusing in the CSS, idk.

File: youtube.html (hunk 1 above)
Comment (derter-menter) [reply youtube.html:12]: yes, as the link is also a slash

**Comment by derter-menter** [comment]
Looks good apart from YouTube page

**Comment by Chiroyce1** [comment]
> Looks good apart from YouTube page

I see no difference at all 

https://user-images.githubusercontent.com/97374054/150633447-1df2b184-c781-482b-af64-4186a0bd735e.mov

**Comment by derter-menter** [comment]
> 

As I added the center tag back, I have no clue what was wrong

**Comment by derter-menter** [comment]
@Chiroyce1 As I added the center tag back, I have no clue what was wrong

**Comment by Chiroyce1** [comment]
Check PR #16

**Pull request merged**